src/cljs/main/broadfcui/page/workspace/create.cljs

```diff
@@ -40,25 +40,28 @@
           (style/create-textfield-hint "Only letters, numbers, underscores, and dashes allowed")
           (style/create-form-label "Description (optional)")
           (style/create-text-area {:style {:width "100%"} :rows 5 :ref "wsDescription"})
-          [:div {:style {:marginBottom "1em"}}
-           [comps/Checkbox
-            {:ref "protected-check"
-             :label "Workspace intended to contain NIH protected data"
-             :disabled? (not= (:protected-option @state) :enabled)
-             :disabled-text (case (:protected-option @state)
-                              :not-loaded "Account status has not finished loading."
-                              :not-available "This option is not available for your account."
-                              nil)}]]
+          [:div {:style {:display "flex"}}
```

Comment:
**Before:** Showed an enabled checkbox if you were a member of the dbGap realm, disabled otherwise
**After:** Shows the dropdown with all authorization domains that you are a member of